ERROR - 2021-06-06 04:24:18 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-06-06 04:24:18 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#7 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-06-06 04:24:23 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-06-06 04:24:23 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#7 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-06-06 04:26:57 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-06-06 04:26:57 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#5 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#7 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#10 {main}
ERROR - 2021-06-06 04:28:41 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
CRITICAL - 2021-06-06 04:28:41 --> Unable to connect to the database.
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(618): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(174): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(518): CodeIgniter\Model->doFind(false, NULL)
#4 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(27): CodeIgniter\BaseModel->find()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-06-06 04:34:39 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:34:46 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:35:40 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:35:41 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(17): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->Dashboard()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-06-06 04:35:42 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(17): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->Dashboard()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-06-06 04:35:43 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-06-06 04:35:43 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(17): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->Dashboard()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-06-06 04:35:44 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-06-06 04:35:48 --> syntax error, unexpected token "echo"
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('Login_View', Array, true)
#2 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(11): view('Login_View')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->index()
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-06-06 04:36:11 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:36:36 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:37:01 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:37:44 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-06-06 04:39:04 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2021-06-06 04:42:05 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('INSERT INTO `ak...')
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ak...')
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ak...')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#9 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2021-06-06 04:42:16 --> Undefined array key 0
#0 C:\xampp\htdocs\ci_pemweb\app\Controllers\Login.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 28)
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->loginUser()
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 {main}
ERROR - 2021-06-06 04:44:10 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('INSERT INTO `ak...')
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ak...')
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ak...')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#9 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2021-06-06 04:49:07 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('INSERT INTO `ak...')
#1 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ak...')
#2 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ak...')
#3 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2263): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ak...', Array, false)
#4 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(278): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\BaseModel.php(758): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\Model.php(705): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci_pemweb\app\Controllers\Register.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Register->daftar()
#9 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\ci_pemweb\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci_pemweb\public\index.php(44): CodeIgniter\CodeIgniter->run()
#12 {main}
